modules/ytp-assets-common/src/less/ckan/upstream_ckan/mixins.less

```diff
@@ -48,9 +48,11 @@
     background-repeat: no-repeat;
 }
 
+// TODO: these do not belong in this file. Only for mixins
 .tag {
     display: inline-block;
-    margin-bottom: 4px;
+    margin-bottom: 2px;
+    margin-bottom: 2px;
```

Comment:
The margin is defined twice?

Reply:
Ah yes the other one was supposed to be margin-top. Will fix.